test/extended/cli/version.go

```diff
@@ -0,0 +1,76 @@
+package cli
+
+import (
+	"fmt"
+	"regexp"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	"strings"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[sig-cli] oc version", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLI("oc-version")
+	ocAdmin := oc.AsAdmin()
+
+	g.It("check kubernetes version", func() {
+		podNames, err := ocAdmin.Run("get").Args("pods", "-l", "app=kube-controller-manager", "-n", "openshift-kube-controller-manager", "-o=jsonpath={.items..metadata.name}").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		podList := strings.Split(podNames, " ")
+
+		imageName, err := ocAdmin.Run("get").Args("pod", "-n", "openshift-kube-controller-manager", podList[0], "-o=jsonpath='{.spec.containers[0].image}'").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		masterNode, err := exutil.GetFirstMasterNode(oc)
```

Comment:
Drop the helper and replace that with:
```
masterNodes, err := oc.AdminKubeClient().CoreV1().Nodes().List(context.Background(), metav1.ListOptions{
		LabelSelector: `node-role.kubernetes.io/master`,
})
o.Expect(err).NotTo(o.HaveOccurred())
```
grab the first element in the list.

Reply:
Or do similar get as you do with `oc get nodes`.

Reply:
Done